packages/library-base/src/commonMain/kotlin/io/realm/internal/RealmObjectHelper.kt

```diff
@@ -133,6 +145,32 @@ object RealmObjectHelper {
         }
     }
 
+    // Consider inlining
+    @Suppress("unused") // Called from generated code
```

Comment:
Already refactoring as part of #596, so doesn't make sense to try to factor to avoid duplication in this PR.